packages/conf-secp256k1/conf-secp256k1.1.0.0/opam

```diff
@@ -11,6 +11,10 @@ depexts: [
   ["dev-libs/libsecp256k1"] {os-distribution = "gentoo"}
   ["domt4/crypto/secp256k1"] {os-distribution = "homebrew" & os = "macos"}
   ["libsecp256k1-git"] {os-distribution = "arch"}
+  ["libsecp256k1" "libsecp256k1-dev"] {os-distribution = "alpine"}
+  ["libsecp256k1" "libsecp256k1-devel"] {os-distribution = "fedora"}
+  ["libsecp256k1" "libsecp256k1-devel"] {os-family = "suse"}
```

Comment:
Hm, opensuse 15.3 is still failing

```
[ERROR] Package conf-secp256k1.1.0.0 depends on the unavailable system packages 'libsecp256k1' and 'libsecp256k1-devel'. You can use `--no-depexts' to attempt installation anyway.
 (score = 100)
2022-05-25 16:09.20: [SKIP] Package not available
```

Reply:
as far as I saw opensuse repos doesn't include dev files for libsecp256k1

Reply:
> as far as I saw opensuse repos doesn't include dev files for libsecp256k1

I've removed the `libsecp256k1-devel` part. Thanks.